bleep-core/src/scala/bleep/bsp/SetupBloopRifle.scala

```diff
@@ -41,17 +41,17 @@ object SetupBloopRifle {
     model.LibraryVersionScheme.VersionScheme.Always,
     model.Dep.Scala("org.scala-lang.modules", "scala-parallel-collections", "always")
   )
-  val versionCombo = model.VersionCombo.Jvm(model.VersionScala.Scala213)
+  val versionCombo = model.VersionCombo.Jvm(model.VersionScala.Scala212)
```

Comment:
They only publish it for 2.12?!

Reply:
It would seem so. I'm not really sure why. Probably it just has not been a priority while the fork was maintained?

Reply:
> They only publish it for 2.12?!

Is that really so @tgodzik?

Also, why are you using `bloop-frontend` when scala-cli is using `bloop-rifle`?

Relevant PRs
 * https://github.com/scalacenter/bloop/pull/2355
 * https://github.com/VirtusLab/scala-cli/pull/3020

Reply:
But looking here, I think that's fine. It's just the internal version of the server. I think we might be able to update to 2.13 now, but didn't have the chance to look.

Reply:
No idea where the frontend dependency comes from, it should all be rifle, just like scala-cli

Reply:
That's the dependency for starting Bloop server itself. Bloop rifle doesn't download anything by itself.

Reply:
This is because bleep bundles bloop instead of expecting the user to have it on PATH right?

Reply:
Doesn't scala-cli and metals have to do something similar? (metals might not need to as it can depend on the build tool to manage it). If the user does not have bloop on PATH you will need bloop-frontend to start the server, no?

Reply:
We never look for Bloop on PATH. If it was already started then we can just use the same artifacts it downloaded, otherwise we just download what is needed and provide classpath to bloop rifle. But yes, the current approach here is the same as Scala CLI and Metals do it.